src/database.ts

```diff
@@ -976,6 +1009,20 @@ class Database extends GrpcServiceObject {
       callback!
     );
   }
+
+  async getRestoreInfo(): Promise<IRestoreInfoTranslatedEnum | undefined> {
+    const [metadata] = await this.getMetadata();
+    return metadata.restoreInfo ? metadata.restoreInfo : undefined;
```

Comment:
Just `return metadata.restoreInfo;` should have the same effect (I think?)

Reply:
Unfortunately I get this error:

```
Type 'TranslateEnumKeys<IRestoreInfo, "sourceType", typeof RestoreSourceType> | null | undefined' is not assignable to type 'TranslateEnumKeys<IRestoreInfo, "sourceType", typeof RestoreSourceType> | undefined'.
  Type 'null' is not assignable to type 'TranslateEnumKeys<IRestoreInfo, "sourceType", typeof RestoreSourceType> | undefined'.
```